README.md

```diff
@@ -321,6 +321,13 @@ Acknowledging a message indicates to the Memphis server to not <br>re-send the s
 message.Ack();
 ```
 
+### Delay the message after a given duration
+Delay the message and tell Memphis server to re-send the same message <br>again to the same consumer group.
```

Comment:
add a note to mention that the message will be redelivered only in case maxMessageDeliveries not reached yet

Reply:
Added a note for the same